main.js

```diff
@@ -1,28 +1,33 @@
-require("dotenv").config()
-const express = require("express")
-const cors = require("cors")
-const bodyParser = require("body-parser")
+require("dotenv").config();
+const express = require("express");
+const cors = require("cors");
+const bodyParser = require("body-parser");
 
-const colectionRouter = require("./routes/kolekcije/index")
-const { port, domain } = require("./config/setup")
-const registracija = require("./routes/users/registracija")
+const colectionRouter = require("./routes/kolekcije/index");
+const { port, domain } = require("./config/setup");
+const registracija = require("./routes/users/registracija");
+const about = require("./routes/users/about");
 
 // Config
-const app = express()
-app.use(cors())
-app.use(bodyParser.urlencoded({ extended: false }))
-app.use(bodyParser.json())
+const app = express();
+app.use(cors());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
 app.use((req, res, next) => {
-  console.log(req.url)
-  next()
-})
+  console.log(req.url);
+  next();
+});
+
+// User routes
+
+app.get("/korisnici/:id", about);
```

Comment:
ovo je ok za sada, ali planiraj sve korisnicke rute pod jednim imenskim prostorom, npr:
```
korisnici/:id
korisnici/registracija
korisnici/login
korisnici/me
korisnici/izmeni-lozinku
```
i slicno. Cisto da ih mozemo odvojiti u zaseban folder, i da imaju svoj prefiks, kao i kolekcije.

Reply:
To sam kontao sledece da pravimo express.Router za korisnike

Reply:
kul, cisto da imas stvari u perspektivi